dataland-frontend/tests/e2e/fixtures/sfdr/SfdrPreparedFixtures.ts

```diff
@@ -40,18 +43,70 @@ export function generateSfdrPreparedFixtures(): Array<FixtureData<SfdrData>> {
  */
 function manipulateFixtureForOneSfdr(input: FixtureData<SfdrData>): FixtureData<SfdrData> {
   input.companyInformation.companyName = "company-with-one-sfdr-data-set";
-  input.t.social!.general!.annualReport = "/test_pdf.pdf";
+  input.t.general.general.referencedReports = undefined;
   return input;
```

Comment:
This prepared fixture seems to be completely useless.  It does not set any setting to a specific one inside of the actual sfdr datset. We can remove it I think.
The component test which uses this prepared fixture should work perfectly fine with a "normal" sfdr fixture.

Reply:
Removed and changed to other fake entry where it was previously used

Reply:
👍